### ISSUE TITLE
Manual: Update output file locations in subdirectories

### ISSUE THREAD

**Issue by bangerth** [geodynamics/aspect#1199]
#1076 moves graphical output files into subdirectories of the output directory, but this is not reflected in the manual in a number of places. For example, searching for `/solution-` yields a number of occurrences that we ought to update.

**Comment by class4kayaker** [comment]
When grepping manual.tex, I found only one actual reference to such a file, and a few references copied from example log files(log.txt and statistics).

Checking the log.txt and statistics file of a current run of simpler_box, I found that they still indicate that the file being written to is output/solution-${N}.

**Comment by bangerth** [comment]
grep for `solution-` instead and you will find quite a number of places. I think we have to rewrite at least part of the section on "Running aspect".

Are you saying that `log.txt` and `statistics` are wrong then? Or are we still writing the `solution-0000.visit` file into the output directory, but the `solution-0000.PPPP.vtu` files into the subdirectory instead?

**Comment by class4kayaker** [comment]
I see an additional output of the `ls` command, and one location where it will be necessary to modify the text to correct the directory being referred to, or adjust what the suggested methods for using Visit are entirely. I did find that I had miscounted, and there were two locations where I had found references to the full relative path. In the other cases it is much more of a judgement call as the files are being referred to in a manner which is still unambiguous.

Yes, I am saying that the `log.txt` and `statistics` files are not correctly indicating the files being written. Given the scale of the change in expected output, this should probably be an entirely separate pull request so as to correctly isolate changes in the tests.
